src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java

```diff
@@ -12,14 +12,62 @@
 public class DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest {
```

Comment:
Could you name this class ` AmountOfTimeBeforeOrAfterCourseCompletionTest`since it contains tests for the ` AmountOfTimeBeforeOrAfterCourseCompletion` class?